app/assets/javascripts/search-result-load-in.js

```diff
@@ -3,12 +3,12 @@ $(document).ready(function() {
   $('.search-results__card').each(function() {
     $(this)
       .css({
-        transitionDelay: (searchResult * 0.05) + 's',
+        transitionDelay: (searchResult * 0.10) + 's',
```

Comment:
Tiny 🐘 : I don't think we need that trailing zero. You could probably just write it as 0.1. Super small thing.